compiler/src/iree/compiler/Codegen/LLVMGPU/test/ROCDL/pipeline_vector_distribute_gfx942.mlir

```diff
@@ -1312,3 +1314,127 @@ module {
 // MEMORY-LABEL: func.func @attention_gather_k
 // MEMORY-COUNT-3: memref.alloc
 // MEMORY-NOT:     memref.alloc
+
+// -----
+
+#translation = #iree_codegen.translation_info<pipeline = LLVMGPUVectorDistribute workgroup_size = [256, 1, 1] subgroup_size = 64>
+#lowering_config = #iree_gpu.lowering_config<{reduction = [0, 0, 0, 0, 0, 512], workgroup = [1, 1, 1, 32, 0, 0]}>
```

Comment:
Can you use partial_reduction instead of reduction here?

Reply:
Also, we should be tiling the outer K2 dimension to number of warps?

Reply:
This is just and example.
Would you be able to spell out the config that you d like tested here ?